core/tests/performance_tests/splash_test.py

```diff
@@ -32,20 +32,20 @@ def test_splash_page_has_less_than_10_megabytes_sent_to_the_client(self):
             self.page_metrics.get_total_page_size_bytes(), 10000000)
 
     def test_splash_page_loads_under_10_seconds(self):
-        self._record_page_timings_for_url(self.SPLASH_URL)
+        self._record_page_timings_for_url(self.SPLASH_URL, session_count=3)
```

Comment:
Is the extra ``session_count=3`` here (and elsewhere) necessary? If the default was fine before, shouldn't it still be now?